### ISSUE TITLE
I don't have Text Processing in menu

### ISSUE THREAD

**Issue by spectre008** [AwesomeTTS/awesometts-anki-addon#221]
I can't find in Tools -> Language Tools: Text Processing

**Comment by lucwastiaux** [comment]
Are you using AwesomeTTS or Language Tools ? they are two different addons.

**Comment by spectre008** [comment]
I'm using Awesome TTS

**Comment by lucwastiaux** [comment]
AwesomeTTS doesn't have this menu item, is there a reason you're looking for this ? you're following a tutorial ? see tutorials for both here: https://languagetools.anki.study/tutorials

**Comment by spectre008** [comment]
Ok, I got it. Well, when I generate audios I just wanna replace "/" to " ". Is there any chance how I can do this?

**Comment by lucwastiaux** [comment]
yes it's possible, do you know how to use text replacement rules in AwesomeTTS ?

**Comment by spectre008** [comment]
No, I don't.
